tests/roles/glance_adoption/defaults/main.yaml

```diff
@@ -1,2 +1,2 @@
-# glance_backend can be 'local' or 'ceph'
-glance_backend: local
+# glance_backend can be 'local', 'ceph' or 'cinder'
+glance_backend: cinder
```

Comment:
I would leave the default to `swift` because in TripleO, if no ceph || any other backend is spefcified,  it's what by default the src cloud uses.
I agree that we might need to pass this variable as an override to the playbook in case we need to test the cinder use case.

Reply:
Ack